packages/graphiql/docs/plugins.md

```diff
@@ -0,0 +1,111 @@
+# Creating GraphiQL Plugins!
+
+> coming soon: `create-graphiql-plugin`! check back for
+
+Developing GraphiQL plugins has always been possible since we introduced them in `graphiql@2`, but the configuration has been a bit tricky
+For this, we've created [`@graphiql/plugin-utils`](../../graphiql-plugin-utils/) to help minimize the configuration footprint, set some conventions, and get you up and running
```

Comment:
```suggestion
Developing GraphiQL plugins has always been possible since we introduced them in `graphiql@2`, but the configuration has been a bit tricky.
For this, we've created [`@graphiql/plugin-utils`](../../graphiql-plugin-utils/) to help minimize the configuration footprint, set some conventions, and get you up and running.
```